File: 1-9/2/Main.java
public class Main {
    public static void main(String[] args) {
        int sum = 0;
        int current = 1;
        int previous = 0;
        int temp;

        while (current <= 4000000) {
            sum = (current % 2 == 0) ? sum + current : sum;

            temp = current;
            current = current + previous;
            previous = temp;
        }

        System.out.println("Sum of even Fibonacci numbers: " + sum);
    }
}

File: 1-9/2/Main.kt
fun main() {
    var sum = 0
    var current = 1
    var previous = 0

    while (current <= 4000000) {
        if (current % 2 == 0) {
            sum += current
        }

        val temp = current
        current += previous
        previous = temp
    }

    println("Sum of even Fibonacci numbers: $sum")
}
